src/controllers/inventory.js

```diff
@@ -132,4 +138,28 @@ export default function () {
       })
   }
   loadMore()
+  
+  const $exportInventory = $('#export-inventory')
+  const downloadCsv = exportData => {
+    const columns = [`${i18n(i19sku)}`, `${i18n(i19name)}`, `${i18n(i19price)}`, `${i18n(i19unitsInStock)}`]
+    const csv = Papa.unparse({
+      data: exportData,
+      fields: columns
+    })
+    const csvData = new window.Blob([csv], {
+      type: 'text/csv;charset=utf-8;'
+    })
+    const csvURL = navigator.msSaveBlob
+      ? navigator.msSaveBlob(csvData, 'download.csv')
+      : window.URL.createObjectURL(csvData)
+    const $link = document.createElement('a')
+    $link.href = csvURL
+    $link.setAttribute('download', 'export-inventory.csv')
+    $link.click()
+    $(`#t${tabId}-loading`).hide()
+  }
+  $exportInventory.click(() => {
+    let exportInventory = datatable.rows().data()
+    downloadCsv(exportInventory)
```

Comment:
```suggestion
    downloadCsv(datatable.rows().data())
```

Se fosse pra declarar uma variável seria uma `const` (não é alterada), mas acho que aqui nem precisa na verdade...